src/components/OktaSignInWidget/index.tsx

```diff
@@ -4,22 +4,29 @@ import React, { useEffect, useRef } from 'react';
 import { useTranslation } from 'react-i18next';
 import OktaSignIn from '@okta/okta-signin-widget';
 
+import Spinner from 'components/Spinner';
+import useOktaSession from 'hooks/useOktaSession';
+
 import './index.scss';
 
 type OktaSignInWidgetProps = {
   onSuccess: (auth: any) => any;
   onError: () => void;
   setError: (value: boolean) => void;
+  className?: string;
```

Comment:
I can't find anywhere that this new prop is used -- can we nix it if it's not needed?